index.html

```diff
@@ -1,15 +1,341 @@
 <!DOCTYPE html>
 <html lang="fr">
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <title>Booki</title>
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <meta name="description" content="Hebergement de rêve à porter de clic" />
-  </head>
-
-  <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
-  </body>
-</html>
+    <script src="https://kit.fontawesome.com/0064da0cf4.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header>
+        <img src="./assets/images/logo/Booki@3x.png" alt="Logo de Booki" />
+        <nav>
+            <ul>
+                <li><a href="#">Hébergements</a></li>
+                <li><a href="#">Activités</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <!--Search bar section-->
+        <section id="search-bar">
+            <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
+            <p>En plein centre ville ou en pleine nature</p>
+            <form method="post" action="#">
+                <label for="city-name"><i class="fas fa-map-marker-alt"></i></label>
+                <input type="text" id="city-name" name="city-name" value="Marseille, France" />
+                <button type="submit">
+                    <span>Rechercher</span>
+                </button>
+            </form>
+        </section>
+        <!--Filtres section-->
+        <section id="filter">
+            <h3>Filtres</h3>
+            <ul>
+                <li><i class="fas fa-money-bill-wave"></i>Economique</li>
+                <li><i class="fas fa-child"></i>Familial</li>
+                <li><i class="fas fa-heart"></i>Romantique</li>
+                <li><i class="fas fa-heart"></i>Animaux autorisés</li>
+            </ul>
+            <p><i class="fa-solid fa-circle-info"></i>Plus de 500 logements songit statust disponibles dans cette ville
+            </p>
+        </section>
+        <!--Section Hébergement & Plus populaires-->
+        <section id="Heberge-populaire">
+            <!--Hébergements-->
+            <div class="hebergement">
```

Comment:
Use section with english id like accomodations

Reply:
indeed i found out it makes more sense when i tried to modified your last comment. 
Please check
Thank you